packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1349,7 +1349,7 @@ export class Profiles extends ProfilesCache {
 
     private async updateBaseProfileFileLogout(profile: IProfileLoaded) {
         const upd = { profileName: profile.name, profileType: profile.type };
-        const config = ImperativeConfig.instance.config;
+        const config = (await this.getProfileInfo()).getTeamConfig();
```

Comment:
I do remember some of that conversation, however I don't think we were ever supposed to use the `ImperativeConfig.instance.config` to begin with since that's more of a CLI driven utility.

Is this why we needed to check for the instance in the `globals.profilescache` ?

Reply:
Reverted until we pull-in the next cli version (7.2.2)